package.json

```diff
@@ -6,14 +6,14 @@
     "start": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy"
+    "deploy": "GIT_USER=tasn docusaurus deploy"
```

Comment:
```suggestion
    "deploy": "docusaurus deploy"
```

Reply:
Wouldn't it mess up the branch deployment though? Also, I can set it manually locally if I ever want that.